### PR TITLE
#215 reservation logic

### PR DESCRIPTION
**Pull request by ebercamargo**
#172 Add method with request paramether list (reservations) in view.py.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#398](https://github.com/timelesslounge/timelessis/issues/398) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#398](https://github.com/timelesslounge/timelessis/issues/398) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/a5569af3-c923-4182-9669-b40f088f7f0e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by emilianodellacasa** [comment]
@ebercamargo Are you sure this PR is for #172? That issue has already been solved

**Comment by emilianodellacasa** [comment]
@ebercamargo Right issue is #215 , please change it in your description and title to avoid confusion

**Comment by vladarefiev** [comment]
@ebercamargo change also in ticket description.

File: timeless/reservations/views.py [hunk 1]
```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
...
 def list_reservations(reservations):
-    """
-        @todo #172:30min Refactor this after the implementation of GenericViews.
-         Take a look at puzzles #134 and #173 where the requirements of generic
-         views are described. Don't forget to cover the generated code with
-         tests
-
-    :param reservations:
-    :return:
+    reservations_result = []
```
Comment (vladarefiev) [inline timeless/reservations/views.py]: @ebercamargo We have a `ListView` generic class, pls use it here. Look at code base there are a lot of examples.

File: timeless/reservations/views.py [hunk 2]
```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
 
 @bp.route("/")
 def list_reservations(reservations):
-    """
-        @todo #172:30min Refactor this after the implementation of GenericViews.
```
Comment (vladarefiev) [inline timeless/reservations/views.py]: @ebercamargo revert this puzzle

File: timeless/reservations/views.py [hunk 3]
```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
...
-        "restaurants/tables/list.html", reservations=reservations
-    )
+    """
+    if not reservations_result:
+        flash('No results found!')
+        return redirect('/')
+    else:
+        reservations_table = Results(reservations_result)
+
+    	return render_template(
```
Comment (vladarefiev) [inline timeless/reservations/views.py]: @ebercamargo leave puzzle for implementing tests for this view

File: timeless/reservations/views.py [hunk 4]
```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
...
-         tests
-
-    :param reservations:
-    :return:
+    reservations_result = []
+    reservations_string = reservations.data['reservations']
+    qry = db_session.query(Reservations).filter(Reservations.id.contains(reservations_string))
+    reservations_result = qry.all()
     """
     flash("List not yet implemented")
```
Comment (vladarefiev) [inline timeless/reservations/views.py]: @ebercamargo dont forget to remove it

**Comment by vladarefiev** [comment]
@ebercamargo Hey, what's up? :)

**Comment by 0crat** [comment]
@vladarefiev[/z](https://www.0crat.com/u/vladarefiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by emilianodellacasa** [comment]
@ebercamargo closed for pr #463

**Comment by 0crat** [comment]
Job [#398](https://github.com/timelesslounge/timelessis/issues/398) is not in the agenda of @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), can't inspect

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#398](https://github.com/timelesslounge/timelessis/issues/398) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request closed without merging**